xiblint/rules/__init__.py

```diff
@@ -4,14 +4,29 @@
 from importlib import import_module
 
 
+class Rule(object):
+    def __init__(self, config):
+        self.config = config
+
+    @staticmethod
```

Comment:
I think often a `check` would actually be an instance method since it wants access to `self.config`, right?

Reply:
yea I guess so, is it a problem to have this difference? I only made it this to avoid pylint warnings, happy to remove it though, this does seem to work:

```py
class Foo(object):
    @staticmethod
    def bar(config):
        print("foo, ", config)

class Baz(Foo):
    def bar(self, config):
        super(Baz, self).bar(config)
        print("baz,", config)

b = Baz()
b.bar("hi")
```

Reply:
Removed it, we can revisit if we ever care about the lint warning